cli/serve/serve.go

```diff
@@ -247,7 +247,9 @@ var endpoints = map[string]func() (http.Handler, error){
 func registerHandlers() {
 	for path, getHandler := range endpoints {
 		log.Debugf("getHandler for %s", path)
-		if handler, err := getHandler(); err != nil {
+		if strings.Contains(conf.Disable, path) {
```

Comment:
Thanks @knackaron. Unfortunately there are some endpoints which are substrings of others (e.g. `crl` and `gencrl`), so this check needs to be refined. Can you instead split `conf.Disable` on commas and do an exact comparison? Something like this should work:
```go
disabled := map[string]struct{}
if conf.Disable != "" {
	for _, endpoint := range strings.Split(conf.Disable, ",") {
		disabled[endpoint] = struct{}{}
	}
}

// ...

if _, ok := disabled[path]; ok {
	log.Infof("endpoint '%s' is explicitly disabled", path)
}
```